### DISCUSSION TITLE
Welcome to vue-csv-import Discussions!

### DISCUSSION THREAD

**Discussion by jgile** [jgile/vue-csv-import#70] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TTeuZ** [comment]
Hi, My name is Paulo Mateus, im developing a new CRM system and i liked so much your library.

We are trying to use vue-csv-import in vue 2.6.

But, when we execute the command `npm install vue-csv-import --save`, the version installed is only compatible with the vue 3, showing us the typical import errors from the new syntax like those:

![image](https://user-images.githubusercontent.com/72257347/128031986-af8d6e80-c2e5-4e4d-b3af-340d41a5ea6c.png)

There is any way tha we can install the vue 2(present in the branch 'vue 2') version by npm? in case not, there is any other options?

Thanks for reading this =D

**Reply by TTeuZ** [reply]
Hello,

looking into your repository, we find out that the version 3.3.5 works in vue 2.6. So, we installed this version in our project and it worked !!
